src/sandbox/algorithms/td_zero/td_zero.py

```diff
@@ -0,0 +1,30 @@
+from typing import Callable
+from sandbox.algorithms.algorithm import Algorithm
+import gym
+import numpy as np
+from sandbox.enviroments.base.env import Environment
+
+from sandbox.enviroments.multi_armed_bandit.policy import DeterministicPolicy, Policy
+from sandbox.value_estimates.tabular_value_estimates import TabularStateValueEstimates
+
+
+class TDZero(Algorithm[TabularStateValueEstimates]):
+
+    def __init__(self, alpha: float, gamma: float, sample_actions: int = 40) -> None:
+        self.alpha = alpha
+        self.gamma = gamma
+        self.sample_actions = sample_actions
+
+    def run(self, n_episodes: int, env: Environment, policy: DeterministicPolicy) -> TabularStateValueEstimates:
+        state_value = TabularStateValueEstimates(env.all_observations(), lambda o: 0 if env.is_terminal_observation(o) else np.random.random())
+        for _ in range(n_episodes):
+            observation = env.reset(seed=42, return_info=False)
+            done = False
+            while not done:
+                actions = np.unique(env.action_space_for_observation(observation).sample() for i in _ range(self.sample_actions))
```

Comment:
I tried to find a smarter way to get all possible actions from state, but it looks like it is the smartest.

`OpenAiGym` represents `actions` as a space from which actions could be sampled, and it makes sense because you will never knew all actions from `observation` if you action state is Continuous.